src/GameFace/Models/StatisticDatapoint.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace GameFace.Models
+{
+    public class StatisticDatapoint
+    {
+        public int id { get; set; }
+        public int idUser { get; set; }
+        public int idStatistic { get; set; }
+        public int idTask { get; set; }
+        public string description { get; set; }
+
+        public virtual Users users { get; set; }
```

Comment:
You might want to add the JsonIgnore attribute here for the navigation properties, but you decide if you need them serialised.